backend/bin/deal-observer-backend.js

```diff
@@ -133,7 +133,14 @@ export const lookUpPayloadCidsLoop = async (makeRpcRequest, getDealPayloadCid, p
     // Maximum number of deals to look up payload CIDs for in one loop iteration
     const maxDeals = 1000
     try {
-      await lookUpPayloadCids(makeRpcRequest, getDealPayloadCid, pgPool, maxDeals)
+      const numOfUnresolvedPayloadsCidsResolved = await lookUpPayloadCids(makeRpcRequest, getDealPayloadCid, pgPool, maxDeals)
+      const numOfUnresolvedPayloadsCids = await countStoredActiveDealsWithUnresolvedPayloadCid(pgPool)
+      if (INFLUXDB_TOKEN) {
+        recordTelemetry('look_up_payload_cids_stats', point => {
+          point.intField('total_unresolved_payload_cids', numOfUnresolvedPayloadsCids)
+          point.intField('unresolved_payload_cids_resolved', numOfUnresolvedPayloadsCidsResolved)
+        })
+      }
```

Comment:
Consistent with my suggestion above:

btw, it shouldn't be `PayloadsCids` but `PayloadCids`. The difference is:
- "Look up payload cids" = Look up the cids of payloads (correct)
- "Look up payloads cids": I don't know what this should mean. Did you mean to say something like "Look up payloads' cids"?

```suggestion
      const numOfPayloadCidsLookedUp = await lookUpPayloadCids(makeRpcRequest, getDealPayloadCid, pgPool, maxDeals)
      const numOfPayloadsWithoutCid = await countStoredActiveDealsWithoutPayloadCid(pgPool)
      if (INFLUXDB_TOKEN) {
        recordTelemetry('look_up_payload_cids_stats', point => {
          point.intField('payloads_without_cid', numOfPayloadsWithoutCid)
          point.intField('payload_cids_looked_up', numOfPayloadCidsLookedUp)
        })
      }
```

Depending on whether you decide for "without" or "missing" in the comment above, please also adjust here

Reply:
Resolved, Unresvoled is a state and look up is the action.  The state "Looked Up" sounds odd to me which is why I did not choose it.

Reply:
The fitting action for resolved and resolved is "to resolve", see comment above.